warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/BulkIngestMapFileLoader.java

```diff
@@ -534,6 +542,18 @@ public void run() {
         log.info("Bulk map file loader shutting down.");
     }
     
+    protected void cleanJobDirectoriesOnStartup() throws IOException {
+        Path[] cleanupDirectories = getJobDirectories(new Path(workDir, jobDirPattern + '/' + CLEANUP_FILE_MARKER));
+        for (int i = 0; i < cleanupDirectories.length; i++) {
+            
+            markSourceFilesLoaded(cleanupDirectories[i]);
```

Comment:
I realize that the markSourceFilesLoaded can supposedly be run by multiple instances concurrently, but how about the runCleanUpScript?  Since that is being passed in, I am thinking this could be a problem.

Reply:
We don't actually use this script option anywhere.  Might be an artifact from a bygone era.  Thinking to remove the logic and option for it since it seems hokey to call a bash script like this anyway. Thoughts?